### ISSUE TITLE
(AWS::Scheduler::ScheduleGroup) Implement `GetAtt` for the `Resource Name`

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/cloudformation-coverage-roadmap#1718]
### Name of the resource

Other

### Resource name

AWS::Scheduler::ScheduleGroup

### Description

Currently `AWS::Scheduler::ScheduleGroup` only supports `Ref` to get the `Resource Name`. Implementing this would be great.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-scheduler-schedulegroup.html

Related to https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/68

**Comment by benkehoe** [comment]
Most resources support name and ARN, one only through `Ref` and one through `GetAtt` (maddeningly it's not consistent which one goes with which). What drives your need to use `GetAtt` instead of `Ref`?

**Comment by r-heimann** [comment]
> Most resources support name and ARN, one only through `Ref` and one through `GetAtt` (maddeningly it's not consistent which one goes with which). What drives your need to use `GetAtt` instead of `Ref`?

I'm going to answer this question with a quote because this has already been answered years ago, !Ref is EVIL

> > Is there value in enabling the !GetAtt function to retrieve any attribute of a resource, rather than the current subset we offer?
> 
> Is +∞ an appropriate amount of value? That would make working with CloudFormation SO much easier. Personally, I avoid !Ref at all costs because I have to interrupt and go check the docs to see what !Ref returns for a specific resource type. Making the returned value explicit with !GetAtt would be YUGE!!!!

**Comment by benkehoe** [comment]
I agree, it's annoying! And with the new resource provider model, the identifiers used for `Ref` _have names already._ But your desire for it isn't specific to this resource, correct? Instead of trying to get CloudFormation to do this on a resource-type-by-resource-type basis, I think this might be better to open an issue (or move #68) on the CloudFormation language discussion repo for a general "resource identifiers should also be named attributes" change https://github.com/aws-cloudformation/cfn-language-discussion

**Comment by r-heimann** [comment]
Still no `!GetAtt` support

**Comment by r-heimann** [comment]
Still no `!GetAtt` support
